python/seldon_core/metadata.py

```diff
@@ -0,0 +1,185 @@
+import os
+import logging
+
+from typing import Union, List, Dict
+
+
+logger = logging.getLogger(__name__)
+
+MODEL_IMAGE = os.environ.get("PREDICTIVE_UNIT_IMAGE")
+
+
+class SeldonInvalidMetadataError(Exception):
+    pass
+
+
+class MetadataTensorValidator:
+    """MetadataTensorValidator class
+
+    Stores and validates metadata_tensor defined as follows
+
+        $metadata_tensor =
+        {
+          "name" : $string,
+          "datatype" : $string,
+          "shape" : [ $number, ... ]
+        }
+
+    """
+
+    def __init__(self, name: str, datatype: str, shape: Union[List[int]]):
+        self.name = str(name)
```

Comment:
Here it would still be name "None"

Reply:
Only if someone call if if `None`, here there is no default value.

Reply:
Good point